src/main/java/org/jenkinsci/plugins/workflow/cps/GroovyClassLoaderWhitelist.java

```diff
@@ -76,21 +68,4 @@ private boolean permits(Class<?> declaringClass) {
         return permits(field.getDeclaringClass()) || delegate.permitsStaticFieldSet(field, value);
     }
 
-    /**
-     * Blocks accesses to some {@link DefaultGroovyMethods}, or any other binary method taking a closure, until JENKINS-26481 is fixed.
```

Comment:
Yes there could be other `Closure`-taking binary methods which we are not translating, but it is too cumbersome to track them. Better to spend time implementing the more general, and precise, [JENKINS-31314](https://issues.jenkins-ci.org/browse/JENKINS-31314).